config/initializers/decidim_overwrites.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+Rails.application.config.to_prepare do
+  Decidim::Surveys::SurveysController.prepend Decidim::Overwrites::Surveys::SurveysController
+  Decidim::Forms::MatrixReadonlyCell.prepend Decidim::Overwrites::Forms::MatrixReadonlyCell
+  Decidim::Forms::AnswerReadonlyCell.prepend Decidim::Overwrites::Forms::AnswerReadonlyCell
```

Comment:
I think we shoud add these files to the md5 checker in spec/lib/overrides_spec.rb